modules/custom-post-types/portfolios.php

```diff
@@ -65,6 +65,14 @@ function __construct() {
 		add_shortcode( 'portfolio',                                                    array( $this, 'portfolio_shortcode' ) );
 		add_shortcode( 'jetpack_portfolio',                                            array( $this, 'portfolio_shortcode' ) );
 
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
```

Comment:
Do we need the check here? Since the filters only exist on wpcom and in Jetpack, maybe we can just add the filters.